ansible/roles-infra/infra-aws-sandbox/defaults/main.yml

```diff
@@ -35,14 +39,14 @@ nuke_sandbox: true
 aws_nuke_binary_url: https://github.com/rebuy-de/aws-nuke/releases/download/v2.11.0/aws-nuke-v2.11.0-linux-amd64
 
 aws_nuke_account_blacklist:
-  - 017310218799 # Master account
-  - 627202319003 # OPENTLC Events
-  - 962799139175 # OPENTLC
-  - 719622469867 # GPE (RHPDS prod)
-  - 550201621713 # openshift BU
-  - 124572886817 # AWS dev/test
-  - 809721187735 # GPTE infrastructure
-  - 384299329206 # GPTE ILT
+  - 017310218799  # Master account
+  - 627202319003  # OPENTLC Events
+  - 962799139175  # OPENTLC
+  - 719622469867  # GPE (RHPDS prod)
+  - 550201621713  # openshift BU
+  - 124572886817  # AWS dev/test
+  - 809721187735  # GPTE infrastructure
+  - 384299329206  # GPTE ILT
```

Comment:
What is this block which adds space to each of these lines?

Reply:
yamllint requires two spaces for inline comments (text before comment.